build.ps1

```diff
@@ -59,6 +59,8 @@ bazel shutdown
 # It isn’t clear where exactly those errors are coming from.
 bazel fetch @nodejs_dev_env//...
 
+bazel clean --expunge
```

Comment:
without this things fall over horribly, I’ll merge it this evening, will reset nodes and then revert it.